config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -319,4 +352,61 @@ periodics:
     - name: config
       configMap:
         name: label-config
-
+# ci-k8sio-cip runs daily, to make sure that the destination GCRs do not deviate
+# away from the intent of the manifest.
+- interval: 24h
+  max_concurrency: 1
+  # This name is the "job name", passed in as "--job=NAME" for mkpj.
+  name: ci-k8sio-cip
+  # Enable Pod Utilities. See https://github.com/kubernetes/test-infra/blob/master/prow/pod-utilities.md.
+  decorate: true
+  extra_refs:
+  # Because of Pod Utilities, we clone the below repo automatically, and, get
+  # dropped into /home/prow/go/src/github.com/kubernetes/k8s.io.
+  - org: kubernetes
+    repo: k8s.io
+    base_ref: master
+  spec:
+    containers:
+    # TODO: Move the official cip image to a more serious location. But it's not
+    # a huge problem because we pin it by the digest.
+    - image: gcr.io/cip-demo-staging/cip@sha256:20190401-v1.0.0-3-g44d6169
```

Comment:
a) are you sure this is a digtest? Looks weirder than I would expect.

Also I would still like to see instructions on how to review the contents of this image

Reply:
Oh my, how embarrassing. Thank you so much for catching this... will fix.

Reply:
Fixed in 66c42a3; PTAL?